### PR TITLE
Manually trigger script for non-hover use cases

### LINKED ISSUE #28: Save Button only loads on page refresh for manually set `data-pin-do` tags
- `gatsby-plugin-pinterest` version: 3.2.2

Problem description:

Save Button does not appear until page refresh if you _don't_ include the `saveButton` attribute in the plugin options (which sets `data-pin-hover`) and instead include a [tag](https://developers.pinterest.com/docs/widgets/save/?) where you want the pin it button to appear like so (which adds the save button for individual images):
```
<a href="https://www.pinterest.com/pin/create/button/" data-pin-do="buttonBookmark">
</a>
```

Suggested solution:
PR incoming

I can create a small reproduction if needed but wanted to submit the issue while it was fresh in my mind.

### PR DESCRIPTION
**Pull request by robinske**
**What**: 
Fixes #28 

Manually triggers PinUtils.build() for non-hover use cases (hover use case works with existing plugin code).

**Why**: 
Without this, the script only works on page refresh due to the way Gatsby loads 3rd party scripts. There is a similar workaround in the [Twitter plugin](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-twitter/src/gatsby-browser.js#L46) and a delay in the [Google Analytics plugin](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-google-analytics/src/gatsby-browser.js#L24)

**Checklist**:

- [x] Documentation
- [x] Tests
- [x] Ready to be merged
      

This feels like a hack since the PinUtils.build() function isn't explicitly documented, but I've tried a lot of different things to make this work and this is the best option I've found. At the very least it's [consistent with the Twitter plugin which is blessed by Gatsby.](https://github.com/gatsbyjs/gatsby/issues/4983#issuecomment-381365516)

### CODE REVIEW

**Comment by robinmetral** [comment]
Looks good @robinske! I think we can just go for this, since as you mentioned, this is what the Twitter plugin does.

One question: could you add a bit about this on the README under "Usage"? I'm thinking we could have a small section explaining how to configure the plugin and use it when we want to only add the pin button to individual images. Happy to do it as well if you'd like me to.

**Comment by robinske** [comment]
Happy to - I think this ties into my comment on #1 about the attributes so some clarification will be useful. I'll get to that sometime this week probably. Thanks for your snappy responses @robinmetral !

**Comment by robinske** [comment]
Updated the docs but for some reason running git commit is now deleting the test file and I have no idea why - I'll have to look into that another time. Let me know if you know why that's happening @robinmetral

**Comment by robinske** [comment]
Updated the docs!

File: README.md [hunk 1]
```diff
@@ -51,6 +53,14 @@ This library has a `peerDependencies` listing for [`gatsby`][gatsby].
 
 ## Usage
 
+Use the `options` to configure the script with
+[available attributes](https://developers.pinterest.com/docs/widgets/save/?#button-style-settings).
+
+Note: not all attributes are supported in the plugin yet.
+[See issues for more details](https://github.com/robinmetral/gatsby-plugin-pinterest/issues).
```
Comment (robinmetral) [inline README.md:60]: Thanks for adding this 🙏  We can also revisit this README once all options are supported. If it gets too long we could also extract docs to another page, or even deploy a small example site to Netlify 🙂

**Comment by robinmetral** [comment]
This looks great, thank you @robinske! 

@all-contributors please add @robinske for code and doc

**Pull request merged**